sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -364,6 +657,1448 @@ def test_get_ycable_port_instance_from_logical_port(self):
+        assert(rc == None)
+    
+  
+    @patch('xcvrd.xcvrd_utilities.port_mapping.PortMapping.logical_port_name_to_physical_port_list', MagicMock(return_value=[0]))
+    @patch('xcvrd.xcvrd_utilities.y_cable_helper.y_cable_wrapper_get_presence', MagicMock(return_value=True))
+    def test_update_appdb_port_mux_cable_response_table_port_instance_none(self):
+        asic_index = 0
+        appl_db = "TEST_DB"
+        logical_port_name = "Ethernet0"
+        read_side = 1
+
+        with patch('xcvrd.xcvrd_utilities.y_cable_helper.y_cable_port_instances') as patched_util:
+            def mock_get():
+                pass
+
+            patched_util.get.return_value = mock_get()
+            port_mapping = PortMapping()
+
+            rc = update_appdb_port_mux_cable_response_table(logical_port_name, port_mapping, asic_index, appl_db, read_side)
+            assert(rc == None)
+
+
+    @patch('xcvrd.xcvrd_utilities.port_mapping.PortMapping.logical_port_name_to_physical_port_list', MagicMock(return_value=[0]))
+    @patch('xcvrd.xcvrd_utilities.y_cable_helper.y_cable_wrapper_get_presence', MagicMock(return_value=True))
+    def test_update_appdb_port_mux_cable_response_table_read_side_none(self):
+        asic_index = 0
+        appl_db = "TEST_DB"
+        logical_port_name = "Ethernet0"
+
+        with patch('xcvrd.xcvrd_utilities.y_cable_helper.y_cable_port_instances') as patched_util:
+            def mock_read_side():
+                pass
+            
+            read_side = mock_read_side()
+
+            patched_util.get.return_value = 0
+            port_mapping = PortMapping()
+
+            rc = update_appdb_port_mux_cable_response_table(logical_port_name, port_mapping, asic_index, appl_db, read_side)
+            assert(rc == None)
+
+
+    @patch('xcvrd.xcvrd_utilities.port_mapping.PortMapping.logical_port_name_to_physical_port_list', MagicMock(return_value=[0]))
+    @patch('xcvrd.xcvrd_utilities.y_cable_helper.y_cable_wrapper_get_presence', MagicMock(return_value=True))
+    def test_update_appdb_port_mux_cable_response_table_active_side_none(self):
+        asic_index = 0
+        appl_db = "TEST_DB"
+        logical_port_name = "Ethernet0"
+        read_side = 1
+
+        with patch('xcvrd.xcvrd_utilities.y_cable_helper.y_cable_port_instances') as patched_util:
+            
+            class PortInstanceHelper():
+                def __init__(self):
+                    self.EEPROM_ERROR = -1
+                
+                # Defining function without self argument creates an exception,
+                # which is what we want for this test.
+                def get_mux_direction():
+                    pass
+
+            patched_util.get.return_value = PortInstanceHelper()
+            port_mapping = PortMapping()
+
+            rc = update_appdb_port_mux_cable_response_table(logical_port_name, port_mapping, asic_index, appl_db, read_side)
+            assert(rc == None)
+
+
+
```

Comment:
formatting, you can use install/use autopep8 with autopep8 --maxlinewidth=120 -i to make things easier

Reply:
Ran autopep8 - latest commit reflects that.